jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -0,0 +1,278 @@
+
+Test splitting is typically set up within a job. In this tutorial, you will modify the `build-and-test` job to define the number of parallel test runs, where the test suites are located, and how tests should be split (in this case, by timing).
+
+== Step one - Add the project
+
+To get started, you need to get the sample app building as a project on CircleCI. 
+
+. https://github.com/ryanpedersen42/circleci-react-test-splitting/fork[Fork the repository] on GitHub, or download the repository then push the project code to your desired VCS provider.
++
+. Open the https://app.circleci.com[CircleCI web app]. Make sure you are in your personal organization (`\https://app.circleci.com/pipelines/<vcs-provider>/<your-vcs-username>`), then navigate to **Projects**.
++
+. Find the sample app on the list of projects, and click **Set Up Project**.
++
+. You will be prompted to select your config.yml file. In the window, select the "Fast: Take me to a config.yml template that I can edit" option. Click **Set Up Project**.
++
+. You will then see a list of sample configurations. Scroll down within the window until you see **Node (Advanced)**, then click **Select**.
++
+image::{{site.baseurl}}/assets/img/docs/test-splitting-sample-configs.png[Sample config for Node]
++
+. You will be taken to the configuration. The project actually uses Yarn, so you will need to edit the following steps so that they use `yarn` instead of `npm`, like so:
++
+[source,yaml]
+----
+steps:
+# Checkout the code as the first step.
+- checkout
+# Next, the node orb's install-packages step will install the dependencies from a package.json.
+# The orb install-packages step will also automatically cache them for faster future runs.
+- node/install-packages:
+    # If you are using yarn, change the line below from "npm" to "yarn"
+    pkg-manager: yarn
+- run:
+    name: Run tests
+    command: yarn test
+----
++
+. After you've made these changes, go ahead and click the **Commit and Run** button. This will commit the changes in a new feature branch called `circleci-project-setup` and trigger a new pipeline. 
++
+image::{{site.baseurl}}/assets/img/docs/test-splitting-first-pipeline.png[Successful pipeline run]
++
+Feel free to take a look at the steps run in the pipeline by expanding the green Success status and opening the `build-and-test` job.
++
+image::{{site.baseurl}}/assets/img/docs/test-splitting-first-setup-steps.png[Steps run successfully within the job]
+
+== Step two - Set up test splitting
+
+If you downloaded a local copy of the code repository, carry out the following steps in your text editor to modify `.circleci/config.yml`. Alternatively, you may edit the project's configuration in the CircleCI web app by selecting a branch, and then clicking the **Edit Config** button.
+
+* In the `build-and-test` job, after the `docker` key, add the `parallelism` key with a value of `5`.
```

Comment:
@kbatuigas I just noticed you have used numbered steps in the first section but then bullets for the other sections. Could you change to use numbered steps throughout? Then good to merge, thanks so much for creating this tutorial!!